lime/lime_image.py

```diff
@@ -70,13 +75,12 @@ def get_image_and_mask(self, label, positive_only=True, hide_rest=False,
                 if np.abs(w) < min_weight:
                     continue
                 c = 0 if w < 0 else 1
-                mask[segments == f] = 1 if w < 0 else 2
+                if not negative_only:
```

Comment:
Is there a reason to use -1, 0, 1 instead of 0, 1, 2? Does -1 work well with skimage functions that take segments?

Reply:
I don't know if it's particularly easy for skimage functions.
In my application I wanted to blend different masks and work with the pixels that were part of a higher number of positive than negative masks. Basic adding and using -1, 0, 1 was somewhat more straightforward ghan 0, 1, 2. If   the latter was chosen with a special reason, then I'd leave it like it was.